.eslintrc

```diff
@@ -9,8 +9,9 @@ extends: eslint:recommended
 root: true
 
 rules:
+  comma-style: [2, "last"]
   eol-last: 2
-  indent: [2, 2]
+  indent: [2, 2, {"VariableDeclarator": 2}]
```

Comment:
I'd prefer a rule to force one var declaration per line. This style follows what the node app currently use and is how the predominant node style guide suggests:
http://nodeguide.com/style.html#variable-declarations